cmake_modules/commonSetup.cmake

```diff
@@ -165,8 +165,8 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
     KAFKA
     COUCHBASEEMBED
     SPARK
-    ECLBLAS
-    EXAMPLEPLUGIN)
+    EXAMPLEPLUGIN
+    MONGODBEMBED)
```

Comment:
Minor: Please shift MONGODBEMBED up one line, so EXAMPLEPLUGIN is the last item.

Reply:
I don't remember removing ECLBLAS. I'll add that back as well.